src/components/IconToggle/IconToggle.visual.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { TPAComponentsProvider } from '../TPAComponentsConfig';
+import { IconToggle } from './';
+
+class IconToggleVisual extends React.Component<any> {
+  static defaultProps = {
+    mobile: false,
+  };
+
+  render() {
+    const { mobile } = this.props;
+
+    return (
+      <TPAComponentsProvider value={{ mobile }}>
```

Comment:
Again, the `TPAComponentsProvider` can be removed, since we don't have a mobile design.
The mobile test below can also be removed.